### ISSUE TITLE
Add a command paramater to shred trashed files (replace with 0's)

### ISSUE THREAD

**Issue by peepo5** [andreafrancia/trash-cli#213]
**Is your feature request related to a problem? Please describe.**
I am frustrated when I sell balloons and the government does not like my balloons so they try to infiltrate my computer to get my secret balloon designs and I think I am safe because I emptied my trash with trash-cli, but with file recovery software, the data is recovered and they have my balloon designs.

**Describe the solution you'd like**
Add a parameter like trash-empty -s (for shred).
Basically it would replace the binary values with zeroes to make sure there is no way to recover files.

**Describe alternatives you've considered**
Doing it manually.
